ITC205_BaseLine_Code/src/Loan.java

```diff
@@ -3,36 +3,36 @@
 import java.util.Date;
 
 @SuppressWarnings("serial")
-public class loan implements Serializable {
+public class Loan implements Serializable {
 	
-	public static enum LOAN_STATE { CURRENT, OVER_DUE, DISCHARGED };
+	public static enum LoanState { CURRENT, OVER_DUE, DISCHARGED };
 	
 	private int ID;
 	private book B;
 	private member M;
 	private Date D;
-	private LOAN_STATE state;
+	private LoanState state;
 
 	
-	public loan(int loanId, book book, member member, Date dueDate) {
+	public Loan(int loanId, book book, member member, Date dueDate) {
 		this.ID = loanId;

```

Comment:
D2: Lines 18 - 21, fix variables to be more meaningful 
ID = loanId
B = book
M = member
D = dueDate